app/config/settings_prod.py

```diff
@@ -1,11 +1,11 @@
-from .settings_base import *  # pylint: disable=wildcard-import, unused-wildcard-import
+from .settings_base import *  # noqa: F403
```

Comment:
I don't like the `noqa: F403` is there no way to disable using some wording instead of ID, that something I like in pylint, it make much more clearer what is disabled